public/CNAME

```diff
@@ -1 +1 @@
-www.excalidraw.com
+excalidraw.com
```

Comment:
probably we can delete that file now.. or we still need it?

Reply:
Yeah, it’s for gh-pages